lr-cli/cortx_setup/commands/cluster/config/component.py

```diff
@@ -34,49 +62,176 @@
+            # Parsing nodes
+            for idx, node in enumerate(nodes):
+                if node == local_fqdn:
+                    nodes[idx] = f"srvnode-1:{node}"
+                else:
+                    nodes[idx] = f"srvnode-{idx+1}:{node}"
+
+            # Config validation
+            if not (CONFIG_PATH.exists() or config_path.exists()):
+                raise ValueError(f"'{config_path}' not found. "
+                                 "Please provide valid config path in command.")
+
+            # Build validation
+            if not target_build:
+                target_build = os.environ.get('CORTX_RELEASE_REPO')
+                kwargs['target_build'] = target_build
+                if not target_build:
+                    raise ValueError("'target_build' is mandatory to bootstrap. "
+                                     "Please provide valid build URL in command.")
+
+            # ISO files validation
+            if kwargs['source'] == 'iso':
+                if not (kwargs['iso_cortx'] or kwargs['iso_os']):
+                    raise ValueError(
+                         "iso single file and iso os file paths are mandatory "
+                         "to bootstrap. Please provide valid paths in command.")
+
+            self.logger.info(
+              "Initial checks done. Follow logs for progress. "
+              f"Starting bootstrap process now with args: {kwargs}"
+            )
+            bootstrap_provisioner.BootstrapProvisioner()._run(**kwargs)
+
+#            # Currently, some changes are needed for this step
+#            post_provisioner.PostProvisioner().run(**kwargs)
+
+            self.logger.debug(
+              "Starting with preparing environment. "
+              "Syncing config data now.."
+            )
+            PillarSync().run()
 
-        # Deploy system, prereqs and cortx components
-        if component_group is None:
-            for component_group in noncortx_components:
-                self.logger.info(f"Deploying {component_group} components on nodes")
+            self.logger.debug("Generating cluster")
+            StatesApplier.apply(
+                  ["components.provisioner.config"], targets=ALL_MINIONS
+            )
+
+            self.logger.debug("Creating service user")
+            create_service_user.CreateServiceUser.run(user="cortxub")
+
+            self.logger.debug("Refreshing machine id on the system")
+            reset_machine_id.ResetMachineId.run()
+
+            self.logger.debug("Refreshing enclosure id on the system")
+            RefreshEnclosureId().run()
+
+            self.logger.debug("Setting up Cluster ID on the system")
+            cluster_id.ClusterId().run()
+
+            self.logger.debug("Encrypting config data")
+            EncryptPillar().run()
+
+            self.logger.debug("Exporting to Confstore")
+            confstore_export.ConfStoreExport().run()
+
+            self.logger.info("Bootstrap done. Proceeding to deploy components..")
```

Comment:
Overall only 2 info msgs - both starting at bootstrap and deploy. so, the user will not be too blank on process.